control_data_collecting_tool/scripts/data_collecting_trajectory_publisher.py

```diff
@@ -133,6 +135,18 @@ class DataCollectingTrajectoryPublisher(Node):
     def __init__(self):
         super().__init__("data_collecting_trajectory_publisher")
 
+        # velocity and acceleration grid
+        self.num_bins = 10
+        self.v_min, self.v_max = 0, 13.8
+        self.a_min, self.a_max = -1, 1
+        self.collected_data_counts = np.zeros((self.num_bins, self.num_bins))
+        self.v_bins = np.linspace(self.v_min, self.v_max, self.num_bins + 1)
+        self.a_bins = np.linspace(self.a_min, self.a_max, self.num_bins + 1)
+        self.fig, self.axs = plt.subplots(2, 1, figsize=(10, 12))
```

Comment:
@kosuke55 

Here, `plt` is being used, but whether `matplotlib` is imported is managed by a flag as shown below.

https://github.com/autowarefoundation/autoware_tools/blob/cfb4cbe0415ab0e648bbb17dc425ca91afebc8e3/control_data_collecting_tool/scripts/data_collecting_trajectory_publisher.py#L40

Therefore, it seems that an error occurs when this flag is set to False.